src/Ajax.js

```diff
@@ -337,19 +337,15 @@ var Ajax = module.exports = kind(
 	* @private
 	*/
 	updateProgress: function (event) {
-		// IE8 doesn't properly support progress events and doesn't pass an object to the
-		// handlers so we'll check that before continuing.
-		if (event) {
-			// filter out 'input' as it causes exceptions on some Firefox versions
-			// due to unimplemented internal APIs
-			var ev = {};
-			for (var k in event) {
-				if (k !== 'input') {
-					ev[k] = event[k];
-				}
+		// filter out 'input' as it causes exceptions on some Firefox versions
+		// due to unimplemented internal APIs
+		var ev = {};
+		for (var k in event) {
+			if (k !== 'input') {
+				ev[k] = event[k];
 			}
-			this.sendProgress(event.loaded, 0, event.total, ev);
 		}
+		this.sendProgress(event.loaded, 0, event.total, ev);
```

Comment:
Did you mean to move this outside the loop?  Was it a mistake to have inside the loop to begin with?